packages/typespec-ts/src/modular/emitModels.ts

```diff
@@ -10,7 +10,9 @@ export function buildModels(
   project: Project,
   srcPath: string = "src"
 ): SourceFile {
-  const modelsFile = project.createSourceFile(`${srcPath}/src/api/models.ts`);
+  const modelsFile = project.createSourceFile(
+    `${srcPath}/src/models/models.ts`
```

Comment:
Not sure if `models/models.ts` is a good filename, but I want to put all those models into one separate non-index.ts file.

Reply:
If I divide it into duplicate files based on operation group name etc, it will potentially have the issue of the same models duplicated copies in different files.

Reply:
Yeah I think putting everything in one place for now is simpler. Not sure what a better name would be since these models could be used for both input and output. I think the generic term for these things are simply `values` but that name is just as generic as `models` anyway.

Reply:
will just keep models/models.ts then :)